cli/types/cypress.d.ts

```diff
@@ -716,15 +728,22 @@ declare namespace Cypress {
      *
      * @see https://on.cypress.io/clock
      * @example
-     *    // your app code
+     *    // in your app code
      *    $('#date').text(new Date().toJSON())
-     *    // from spec file
-     *    const now = new Date(2017, 3, 14).getTime() // March 14, 2017 timestamp
+     *    // in the spec file
+     *    // March 14, 2017 timestamp or Date object
+     *    const now = new Date(2017, 3, 14).getTime()
```

Comment:
This is actually April 14. 😅 Love JS dates.
```suggestion
     *    // March 14, 2017 timestamp or Date object
     *    const now = new Date(2017, 2, 14).getTime()
```